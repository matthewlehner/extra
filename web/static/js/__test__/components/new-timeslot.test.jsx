// @flow

import React from "react";
import renderer from "react-test-renderer";

import NewTimeslot from "components/new-timeslot";

describe("NewTimeslot component", () => {
  it("renders the component", () => {
    const tree = renderer.create(
      <NewTimeslot
        addTimeslot={jest.fn()}
        scheduleId={"1"}
        collections={[]}
        recurrenceType={{ enumValues: [] }}
      />
    );
    expect(tree).toMatchSnapshot();
  });
});
