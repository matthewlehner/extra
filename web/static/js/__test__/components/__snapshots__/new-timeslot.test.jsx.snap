// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewTimeslot component renders the component 1`] = `
<button
  onClick={[Function]}
>
  Add new time slot
</button>
`;
